sdk/evaluation/azure-ai-evaluation/tests/unittests/test_lazy_imports.py

```diff
@@ -0,0 +1,127 @@
+"""Tests for lazy import behavior of optional dependencies."""
+
+import sys
+import unittest
+from io import StringIO
+import importlib
+
+
+class TestLazyImports(unittest.TestCase):
+    """Test lazy import behavior for optional dependencies."""
+
+    def test_no_messages_during_module_import(self):
+        """Test that no messages are printed when importing the main module."""
+        # Capture stderr to check for unwanted messages
+        captured_stderr = StringIO()
+        original_stderr = sys.stderr
+        sys.stderr = captured_stderr
+
+        try:
+            # Test imports that would normally fail with missing dependencies
+            # Since we can't easily control the dependency availability in the test environment,
+            # we test the lazy import setup directly
+
+            # This should not print any messages during setup
+            _lazy_imports = {}
+            _patch_all = []
+
+            def _create_lazy_import(class_name, module_path, dependency_name):
+                """Create a lazy import function for optional dependencies."""
+
+                def lazy_import():
+                    try:
+                        module = __import__(module_path, fromlist=[class_name])
+                        cls = getattr(module, class_name)
+                        _patch_all.append(class_name)
+                        return cls
+                    except ImportError:
+                        raise ImportError(
+                            f"Could not import {class_name}. Please install the dependency with `pip install {dependency_name}`."
+                        )
+
+                return lazy_import
+
+            # Setting up lazy imports should not print any messages
+            _lazy_imports["AIAgentConverter"] = _create_lazy_import(
```

Comment:
Can you remove this check from test since this package is always installed as part of dev requreiments